### ISSUE TITLE
How are PriorBoxLayer and MultiBoxLossLayer implemented?

### ISSUE THREAD

**Issue by alex-birch** [seann999/ssd_tensorflow#7]
Hi,

I'm quite new to tensorflow - could you please provide an explanation how you've implemented SSD without the 2 custom layers defined in Caffe - PriorBoxLayer and MultiBoxLossLayer?

From my (limited) understanding - MultiBoxLossLayer is needed for training and backpropagation connecting the ground truth boxes to intermediate layers and loss calculation. PriorBox is used in the forward pass only to generate boxes at the predefined sizes / aspect ratios.

Thanks
